zenodo/modules/github/views/github.py

```diff
@@ -219,3 +220,22 @@ def sync_repositories():
     )
 
     return render_template("github/index.html", **ctx)
+
+
+@blueprint.route("/latestdoi/<int:user_id>/<path:repository>", methods=["GET"])
+@ssl_required
+def latest_doi(user_id, repository):
```

Comment:
Sorry, I should have been clear in the previous message. Can you move it to ``zenodo.modules.github.views.badge`` instead?